### PR TITLE
Integrate sessions API at event/sessions route

### LINKED ISSUE #639: Integrate API at the events/<event_id>/sessions route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

image

I will work on this

### PR DESCRIPTION
**Pull request by harshitagupta30**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Integrate sessions API at event/sessions route

Fixes #639

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-651-open-event-frontend-fossasia.surge.sh

**Comment by CosmicCoder96** [comment]
@harshitagupta30 Which event should I go to in the deployment link to view some sample sessions in the table ?

**Comment by harshitagupta30** [comment]
@CosmicCoder96 @geekyd @sumedh123 You can check this for the event hackathon event id : 105, identifier : 82e5dff3

**Comment by niranjan94** [comment]
Deployment link: https://pr-651-open-event-frontend-fossasia.surge.sh

**Comment by harshitagupta30** [comment]
@CosmicCoder96 https://pr-651-open-event-frontend-fossasia.surge.sh/events/82e5dff3/sessions/

**Comment by harshitagupta30** [comment]
@CosmicCoder96 @sumedh123 @geekyd Please review this :+1:

File: app/templates/components/ui-table/cell/events/view/sessions/cell-session-state.hbs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<span>
+  {{#if (eq record.state "accepted")}}
```
Comment (geekyd) [inline app/templates/components/ui-table/cell/events/view/sessions/cell-session-state.hbs]: @harshitagupta30 Please use yellow colour for `confirmed` & `pending.`

**Comment by harshitagupta30** [comment]
@geekyd Updated! @CosmicCoder96 @sumedh123 Please review :)

**Comment by sumedh123** [comment]
@harshitagupta30 Please check the travis build.

**Comment by niranjan94** [comment]
Deployment link: https://pr-651-open-event-frontend-fossasia.surge.sh

**Comment by harshitagupta30** [comment]
@sumedh123 done!

**Comment by sumedh123** [comment]
@harshitagupta30 The deployment link is giving 404.

**Comment by harshitagupta30** [comment]
@sumedh123 I am not getting any such error. @CosmicCoder96 @geekyd Can you please check ?

**Comment by harshitagupta30** [comment]
I can also see the 404 error (Not Found). As far I researched this is due to something that is unavailable on server which I guess is not in frontend's hand. Can someone please more clarify this?

**Pull request merged**